services/static-webserver/client/source/class/osparc/data/Resources.js

```diff
@@ -1301,23 +1301,13 @@ qx.Class.define("osparc.data.Resources", {
         }, this);
 
         res.addListenerOnce(endpoint + "Error", e => {
-          let message = null;
-          let status = null;
+          const status = e.getRequest().getStatus();
+          let message = null; // required
+          let errorType = null; // optional
           if (e.getData().error) {
             const errorData = e.getData().error;
-            const logs = errorData.logs || null;
-            if (logs && logs.length) {
-              message = logs[0].message;
-            }
-            const errors = errorData.errors || [];
-            if (message === null && errors && errors.length) {
-              message = errors[0].message;
-            }
-            status = errorData.status;
-          } else {
-            const req = e.getRequest();
-            message = req.getResponse();
-            status = req.getStatus();
```

Comment:
All this was here to handle all kind of different responses coming from the backend. Are you sure your cleanup is still able to handle them?